### ISSUE TITLE
Sort icon is reversed in most tree views

### ISSUE THREAD

**Issue by geoff-bbc** [odoo/odoo#112257]
Impacted versions: 14, 15, 16

Steps to reproduce: Navigate to your favorite tree view. Look at the sort order of the data in the sorted column and compare it to the sort order triangle icon in the header.

Current behavior: The down arrow displays when data is sorted ASC and vice versa

Expected behavior: The down arrow should display when data is sorted DESC and vice versa

Here are some examples across versions and apps:

<img width="156" alt="SNAG_2-7-2023_3 19 28 PM" src="https://user-images.githubusercontent.com/59264352/217696340-249680e2-28dc-415b-b535-7117474734ec.png">
<img width="73" alt="SNAG_2-8-2023_5 54 11 PM" src="https://user-images.githubusercontent.com/59264352/217697129-49c09b73-e2aa-4b42-a8dc-c6cf361bfebd.png">
<img width="95" alt="SNAG_2-8-2023_5 54 42 PM" src="https://user-images.githubusercontent.com/59264352/217697135-3ab96b88-a012-4294-b7bc-970055f9ef13.png">

**Comment by geoff-bbc** [comment]
This would seem to be a straightforward issue to fix and remove a defect that is seen thousands of times per hour! Any comments? I'm an implementation partner, not a developer so I'm not the right person to start working on the fix.

**Comment by pedrobaeza** [comment]
Check this thread: https://github.com/odoo/odoo/issues/68318

**Comment by geoff-bbc** [comment]
Thanks @pedrobaeza - good to know the history, I hadn't found that issue.

I also believe the issue I've raised here is now fixed in my current version of v16, so I am happy. I think it's good UX to match pervasive UI patterns.

Closing...

![SNAG_2-24-2023_11 21 58 AM](https://user-images.githubusercontent.com/59264352/221272255-75253dfd-7f6a-45be-9c02-eeeee5f8af1b.png)
![SNAG_2-24-2023_11 21 42 AM](https://user-images.githubusercontent.com/59264352/221272258-f98cd10b-3f30-4eb4-86bf-f8aa04c217b8.png)
